projects/systelab-translate/.eslintrc.json

```diff
@@ -0,0 +1,61 @@
+{
+  "extends": "../../.eslintrc.json",
```

Comment:
It extends from this file, but the content is quite similar. Can we clean the content on this file?

Reply:
This files have been created by an automatic migration process. But yes, I will check it and remove all duplicities (if possible). Also for the rest of the libraries (preferences, components, charts and login)

Reply:
Removed duplicities